pkg/simplewal/simplewal.go

```diff
@@ -1,9 +1,3 @@
-/*
-Copyright IBM Corp. All Rights Reserved.
-
-SPDX-License-Identifier: Apache-2.0
-*/
-
```

Comment:
I guess it is. I had a tendency to leave it in files that did not fundamentally change since my time at IBM, but now basically all the files contain a mix of IBM and non-IBM code. Will remove.

Reply:
Yes it is.

Reply:
Actually, just realized that I got it all wrong, thinking you were suggesting to remove it, while you were suggesting to keep it. I must be color blind... 🙄 (I don't remember removing it and it must have happened by accident, probably when copying code from another version of the file. That's why I assumed you suggesting removal.)
Let's keep it then, to be on the safe side.

Reply:
Yes, I think it is better to keep the original copyright statement.